src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory_Methods.vb

```diff
@@ -163,6 +163,20 @@ Namespace Microsoft.CodeAnalysis.Operations
                 isImplicit:=isImplicit OrElse argument Is Nothing)
         End Function
 
+        Private Function CreateReceiverOperation(node As BoundNode, symbol As ISymbol) As IOperation
+            If node Is Nothing OrElse node.Kind = BoundKind.TypeExpression Then
+                Return Nothing
+            End If
+
+            If symbol IsNot Nothing AndAlso
+               node.WasCompilerGenerated AndAlso
+               symbol.IsStatic Then
```

Comment:
>symbol.IsStatic [](start = 15, length = 15)

I think we should also check if node is BoundKind.MeReference.

Reply:
We also have to check if it's WithLValueExpressionPlaceholder, but yes, I can do this.

---
In reply to: [149247573](https://github.com/dotnet/roslyn/pull/22972#discussion_r149247573) [](ancestors = 149247573)